include/picongpu/particles/atomicPhysics/FillHistogram.hpp

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2013-2021 Axel Huebl, Felix Schmitt, Heiko Burau, Rene Widera,
+ *                     Richard Pausch, Alexander Debus, Marco Garten,
+ *                     Benjamin Worpitz, Alexander Grund, Sergei Bastrakov,
+ *                     Brian Marre
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+#include "picongpu/particles/atomicPhysics/GetRealKineticEnergy.hpp"
+
+#include <pmacc/mappings/kernel/AreaMapping.hpp>
+#include <pmacc/traits/GetNumWorkers.hpp>
+#include <pmacc/type/Area.hpp>
+
+#include <cstdint>
+
+
+// debug only
+#include <iostream>
```

Comment:
not required if the dead code in this file is deleated.

Reply:
done